observability/cluster_observability_operator/ui_plugins/dashboard-ui-plugin.adoc

```diff
@@ -0,0 +1,24 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="dashboard-ui-plugin"]
+= Dashboard UI plugin
+include::_attributes/common-attributes.adoc[]
+:context: dashboard-ui-plugin
+
+toc::[]
+
+:FeatureName: The Cluster Observability Operator
+include::snippets/technology-preview.adoc[leveloffset=+2]
+
+The dashboard UI plugin supports enhanced dashboards in the OpenShift web console at *Observe* -> *Dashboards* . You can add other Prometheus datasources from the cluster to the default dashboards, in addition to the in-cluster datasource. This results in a unified observability experience across different data sources.
+
+The plugin searches for datasources from `ConfigMap` resources in the `openshift-config-managed` namespace, that have the label `console.openshift.io/dashboard-datasource: 'true'`. 
+
+include::modules/coo-dashboard-ui-plugin-install.adoc[leveloffset=+1]
+
+include::modules/coo-dashboard-ui-plugin-configure.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+[id="additional-resources_{context}"]
+== Additional resources
+
+* See how to link:https://github.com/openshift/console-dashboards-plugin/blob/main/docs/add-datasource.md[add a new datasource] in the link:https://github.com/openshift/console-dashboards-plugin[console-dashboards-plugin] GitHub repository.
```

Comment:
Again, I would double check that you can link to GH because we are not supposed to according to the [guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-to-external-websites:~:text=of%20the%20URL.-,Important,-You%20cannot%20link).

Reply:
I think anything inside github.com/openshift/ is fine - I do need exceptions elsewhere for Korrel8r, but these ones are ok I believe